### ISSUE TITLE
Add possibility to add logos/pictures for publications

### ISSUE THREAD

**Issue by literarymachine** [hbz/oerworldmap#1505]
Moved here from https://github.com/hbz/oerworldmap-ui/issues/285:

Whith the new entry view this would be better, because the place for the logo is so prominent.

**Comment by trugwaldsaenger** [comment]
I agree, that this would look much better. But what should be included here? In some cases we could use the cover of a publication. But what if a cover is not available?
